<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QZuesTestClass</class>
 <widget class="QMainWindow" name="QZuesTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QZuesTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QZuesLineEdit" name="txtZues"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnReset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QZuesListView" name="listView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 50pt &quot;Adobe Arabic&quot;;
color:red;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QZuesLabel" name="lblZues">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid red;
color: rgb(255, 170, 0);
font: 75 14pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabelTextLabelTextLabelTextLabelTextLabelTextLabelTextLabelTextLabelTextLabelTextLabelTextLabelTextLabelTextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QZuesAnimationStackWidget" name="stkTest" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:1px solid green;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNextPage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QZuesLabel</class>
   <extends>QLabel</extends>
   <header>qzueslabel.h</header>
  </customwidget>
  <customwidget>
   <class>QZuesLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qzueslineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QZuesListView</class>
   <extends>QListView</extends>
   <header>qzueslistview.h</header>
  </customwidget>
  <customwidget>
   <class>QZuesAnimationStackWidget</class>
   <extends>QWidget</extends>
   <header>qzuesanimationstackwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qzuestest.qrc"/>
 </resources>
 <connections/>
</ui>
